lab1.md

```diff
@@ -53,6 +53,79 @@ You can follow the labs whilst using Kubernetes, but you may need to make some s
 
 If using a `NodePort` then the gateway address for the OpenFaaS CLI is normally http://IP_ADDRESS:31112/
 
+You can choose different options to run Kubernetes on your machine.
```

Comment:
Most of the instructions for Swarm/K8s are quite brief and work well mixed-in the same file, but Lab 1 looks very complicated if all I want is Swarm in a workshop.

What ideas do we have to make the experience better for Lab 1?

Reply:
It's now separated to Lab 1a and Lab 1b